bench-vortex/src/statpopgen/builder.rs

```diff
@@ -0,0 +1,283 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use std::sync::Arc;
+
+use arrow_array::builder::{
+    ArrayBuilder, BooleanBuilder, Float32Builder, Int32Builder, ListBuilder, StringBuilder,
+    UInt64Builder,
+};
+use arrow_array::{ArrayRef, RecordBatch};
+use arrow_schema::ArrowError;
+use itertools::Itertools as _;
+use noodles_vcf::header::record::value::map::info::{Number, Type};
+use noodles_vcf::record::Info;
+use noodles_vcf::variant::record::info::field::Value;
+use noodles_vcf::{Header, Record};
+use vortex::error::{VortexExpect as _, VortexResult, vortex_err};
+use vortex::utils::aliases::hash_map::HashMap;
+use vortex::utils::aliases::hash_set::HashSet;
+
+use super::vcf_value_conversions::*;
+use crate::statpopgen::schema::SCHEMA;
+
+#[allow(dead_code)]
+#[allow(non_snake_case)]
+#[derive(Default)]
+pub struct GnomADBuilder<'a> {
+    pub CHROM_builder: StringBuilder,
```

Comment:
I wonder if it'd be nice to add a bit of context for ppl that are less familiar with the domain. 

```
Clauded the following:


    // Standard VCF columns - always present
    pub CHROM_builder: StringBuilder,      // Chromosome name
    pub POS_builder: UInt64Builder,        // Genomic position  
    pub ID_builder: StringBuilder,         // Variant ID
    pub REF_builder: StringBuilder,        // Reference allele
    pub ALT_builder: ListBuilder<StringBuilder>, // Alternate alleles (can be multiple)
    pub QUAL_builder: Float32Builder,      // Quality score
    pub FILTER_builder: ListBuilder<StringBuilder>, // Filter flags

etc..
```

Reply:
Done.